idea/src/org/jetbrains/kotlin/idea/intentions/ConvertLambdaToReferenceIntention.kt

```diff
@@ -137,6 +151,7 @@ open class ConvertLambdaToReferenceIntention(textGetter: () -> String) : SelfTar
             if (lambdaValueParameterDescriptors.size < explicitReceiverShift + callableExpression.valueArguments.size) return false
             val resolvedCall = callableExpression.getResolvedCall(context) ?: return false
             resolvedCall.valueArguments.entries.forEach { (valueParameter, resolvedArgument) ->
+                if (resolvedArgument is DefaultValueArgument && languageVersion >= LanguageVersion.KOTLIN_1_4) return@forEach
```

Comment:
This too should be replaced with the `LanguageFeature.FunctionReferenceWithDefaultValueAsOtherType` flag check. I would advise to extract the flag check to some variable and reuse it twice